addons/docs/src/frameworks/ember/jsondoc.js

```diff
@@ -29,6 +36,9 @@ export const extractArgTypes = (componentName) => {
 
 export const extractComponentDescription = (componentName) => {
   const json = getJSONDoc();
+  if (!(json && json.included)) {
+    return {};
```

Comment:
should this be `null`?

Reply:
yeah I looked at the other frameworks and they seem to be returning null for "there's no value, sorry"